opencga-catalog/src/main/java/org/opencb/opencga/catalog/db/mongodb/JobMongoDBAdaptor.java

```diff
@@ -294,6 +293,45 @@ OpenCGAResult<Object> privateUpdate(ClientSession clientSession, Job job, Object
         return endWrite(tmpStartTime, 1, 1, events);
     }
 
+    @Override
+    public OpenCGAResult<ExecutionTime> executionTimeByMonth(Query query)
+            throws CatalogDBException, CatalogParameterException, CatalogAuthorizationException {
+        long startTime = startQuery();
+
+        Bson bsonQuery = parseQuery(query, QueryOptions.empty());
+        List<Bson> aggregation = new ArrayList<>();
+        aggregation.add(Aggregates.match(bsonQuery));
+        aggregation.add(Aggregates.project(Projections.fields(
+                Projections.computed("date", "$" + PRIVATE_MODIFICATION_DATE),
+                Projections.computed("difference", new Document("$toDouble",
```

Comment:
What about running jobs?

Reply:
They will be considered when the execution finishes.